test/tst_input.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+// A class to hold test data, an event type/key/modifiers
```

Comment:
an -> and?

Reply:
I did meant to use "An". "An event (type/key/modifier) and a matching neovim input string"